### ISSUE TITLE
Intel Arc GPU doesn't show load%

### ISSUE THREAD

**Issue by ToastedWookiee** [flightlessmango/MangoHud#1078]
**Describe the bug**
Shows 0% load instead of pulling % from the Render/3D info of intel_gpu_top.

**List relevant hardware/software information**
- EndeavourOS
- MangoHud git: dc1761e
- Intel Arc A770

I know we have limited info from intel_gpu_top, no power or temp stats, but it looks like MangoHud is set to pull load % from the "Render/3D" statistic. 
In the source its looking to pull from the "Render/3D/0" line, my output doesn't have this but just "Render/3D" leaving it as 0%
Frequency does work.

```
{
        "period": {
                "duration": 500.344520,
                "unit": "ms"
        },
        "frequency": {
                "requested": 617.574466,
                "actual": 533.632306,
                "unit": "MHz"
        },
        "interrupts": {
                "count": 481.668111,
                "unit": "irq/s"
        },
        "rc6": {
                "value": 5.328041,
                "unit": "%"
        },
        "imc-bandwidth": {
                "reads": 1960.822377,
                "writes": 633.744355,
                "unit": "MiB/s"
        },
        "engines": {
                "Render/3D": {
                        "busy": 10.876027,
                        "sema": 0.000000,
                        "wait": 0.000000,
                        "unit": "%"
                },
                "Blitter": {
                        "busy": 0.000000,
                        "sema": 0.000000,
                        "wait": 0.000000,
                        "unit": "%"
                },
                "Video": {
                        "busy": 0.000000,
                        "sema": 0.000000,
                        "wait": 0.000000,
                        "unit": "%"
                },
                "VideoEnhance": {
                        "busy": 0.000000,
                        "sema": 0.000000,
                        "wait": 0.000000,
                        "unit": "%"
                },
                "Compute": {
                        "busy": 0.000000,
                        "sema": 0.000000,
                        "wait": 0.000000,
                        "unit": "%"
                }
        }
}
```

**Comment by flightlessmango** [comment]
Does this fix it be1e94f008f3de9bdb94641f2c9c6a1d9fd1c434?

**Comment by flightlessmango** [comment]
made a typo in last, fixed here 5d93e484ce0bb16b573c71842937eed274202612

**Comment by ToastedWookiee** [comment]
Yes it does.
